requirements.txt

```diff
@@ -4,3 +4,4 @@ paramiko
 pytest-testinfra
 pytest-durations
 pyyaml
+obsah @ git+https://github.com/theforeman/obsah
```

Comment:
Let's create a release so we don't need to and `setup.py` can simply point to the desired version.

Reply:
1.1.0 was released
```suggestion
```